signac/contrib/migration/__init__.py

```diff
@@ -5,102 +5,135 @@
 
 import os
 import sys
-from contextlib import contextmanager
+import warnings
 
 from filelock import FileLock
 from packaging import version
 
-from ...common.config import get_config
+from ...common.config import get_config, load_config
 from ...version import SCHEMA_VERSION, __version__
-from .v0_to_v1 import migrate_v0_to_v1
+
+# To be uncommented when switching to version 2.
+# from .v0_to_v1 import _load_config_v1, _migrate_v0_to_v1
```

Comment:
This question is now moot because I've made this change now and structured the code so that we'll always make the change whenever the schema version is updated. There's no reason to continue using the primary config API here, we can just always use the migration-specific config APIs.